packages/react-core/src/components/Modal/__snapshots__/ModalBox.test.js.snap

```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ModalBox Test 1`] = `
+.pf-c-modal-box {
+  display: flex;
+  z-index: 600;
+  flex-direction: column;
+  max-width: 35rem;
+  min-height: calc(undefined + undefined + undefined + undefined + undefined + undefined + calc(1rem * 4 * 1.5));
```

Comment:
These coming back as undefined worry me a little.  I will look into why.  It is either the tool or core variables not working.